indico/queries/submission.py

```diff
@@ -0,0 +1,85 @@
+import json
+from typing import Dict, List, Union
+
+from indico.client.request import GraphQLRequest
+from indico.filters import SubmissionFilter
+from indico.queries import JobStatus
+from indico.types import Job, Submission
+
+
+class ListSubmissions(GraphQLRequest):
+    """
+    Query to list Submissions in the Indico Platform, filtered using optional query arguments.
+    """
+
+    query = """
+        query ListSubmissions($submissionIds: [Int], $workflowIds: [Int], $filters: SubmissionFilter, $limit: Int){
+	        submissions(submissionIds: $submissionIds, workflowIds: $workflowIds, filters: $filters, limit: $limit){
+                submissions {
+                    id
+                    datasetId
+                    workflowId
+                    status
+                    inputFile
+                    inputFilename
+                    resultFile
+                }
+            }
+        }
+    """
+
+    def __init__(
+        self,
+        submission_ids: List[int] = None,
+        workflow_ids: List[int] = None,
+        filters: Union[Dict, SubmissionFilter] = None,
```

Comment:
we could just accept a SubmissionFilter here, forcing the `filters` to go through validation, maybe?

Reply:
Nic had suggested leaving it flexible for now -- I can request a final rewview from him after I implement your changes.

Reply:
nah, that's totally fair then. He's got final say :) I think in that case, maybe you'd want to do what you did before? where you created the SubmissionFilter object - to still be able to run those validation checks? totally up to you

Reply:
that's a fair point

Reply:
My original idea here was to have known fields on a class like SubmissionFilter that would have type completion and the like and the dict alternative here was so if we had not yet added a field to the client it would be available to call (backup). The validation here prevents unknown fields which goes against my intent.

Reply:
Can you elaborate on this @nkassis , I'm not following 

```
 and the dict alternative here was so if we had not yet added a field to the client it would be available to call (backup)
```

Reply:
- @nkassis, so you're suggesting we don't validate in the client, just let Fog do it?